owncloudApp/src/main/java/com/owncloud/android/shares/presentation/ShareActivity.kt

```diff
@@ -114,7 +112,7 @@ class ShareActivity : FileActivity(), ShareFragmentListener {
         setContentView(R.layout.share_activity)
 
         // Set back button
-        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
```

Comment:
What happens if supportActionBar is null?

Reply:
The arrow to go back would not appear in the action bar but you could still use the back button in the Android navigation bar.